talks/manuelsanchez.md

```diff
@@ -0,0 +1,26 @@
+# Developing plugins with Semantic Kernel and Azure OpenAI
+
+## Authors
+
+Manuel Sánchez Rodríguez
+
+## Presenter
+
+Manuel Sánchez Rodríguez
+
+## Abstract (200 words max.) 
+
+This session is designed for developers who want to learn how to create plugins using Semantic Kernel and Azure OpenAI. You will learn how to create plugins that can be used across all of the major AI apps and services like ChatGPT, Bing, and Microsoft 365. This will allow you to increase the reach of your AI capabilities without rewriting your code.
```

Comment:
Thanks, Manuel, but Databeers is not really suited for this kind of session. We're looking for fast (as in 10 minutes) talk that highlight some aspect of data manipulation or visualization or machine learning.

Reply:
We could check the available slots in the workshop space, in case you're interested.